app/models/null/leaderboard_row.rb

```diff
@@ -0,0 +1,36 @@
+module Null
+  class LeaderboardRow
+    attr_reader :player
+
+    def initialize(player)
+      @player = player
+    end
+
+    def position
+    end
+
+    def played
+      0
+    end
+
+    def corp_wins
+      0
+    end
+
+    def runner_wins
+      0
+    end
+
+    def participation_points
+      0
+    end
+
+    def achievement_points
+      0
+    end
+
+    def points
+      0
+    end
+  end
+end
```

Comment:
Huh. Good one Johno, not seen the `Null Object` pattern implemented like this before: I'm more accustomed to seeing the pattern implemented as a nested class, ala `LeaderboardRow::Null`.

I'm absorbing Rails as I read+hack along with `desperado`; looks good so far, I'm learning a lot, cheers!